docs/help/_posts/2010-05-02-throwing-exceptions.markdown

```diff
@@ -13,6 +11,25 @@ ICalculator calculator;
 ```
 -->
 
+The `Throws` and `ThrowsAsync` helpers in the namespace NSubstitute.ExceptionExtensions could be use d
+
+```csharp
+//For non-voids:
+calculator.Add(-1, -1).Throws(new Exception()); // Or .Throw<Exception>() - don't use Throw*s*
```

Comment:
**question:** the "don't use Throws" is related to the question on Throws vs Throw? I think it would need to be `Throws` in this case as it isn't chained from a `WhenCalled`?